app/controllers/teams_controller.rb

```diff
@@ -39,6 +39,17 @@ def update
     change_description(@team, :team)
   end
 
+  # GET /teams/1/typeahead/%QUERY
+  def typeahead
+    authorize @team
+    @query = params[:query]
+    matches = User.search_from_query(@team.member_ids,
+                                     "#{@query}%").map { |user| { username: user } }
```

Comment:
Bootstrap Typeahead for Rails (https://github.com/Nerian/bootstrap-typeahead-rails) uses typeahead 0.10.5.1 and for this version it is necessary to build objects. Should I switch to a newer version of typeahead? (0.11.1)